code/_helpers/items.dm

```diff
@@ -0,0 +1,9 @@
+//Prevents robots dropping their modules.
+/proc/dropsafety(var/atom/movable/A)
+	if (istype(A.loc, /mob/living/silicon))
```

Comment:
There's a helper for this, defined here: https://github.com/Aurorastation/Aurora.3/blob/master/code/_helpers/unsorted.dm#L1209